templates_ark/example_cell_clustering.ipynb

```diff
@@ -77,10 +77,12 @@
     "base_dir = \"../data/granulomaCohort_allData\"\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    pixel_output_dir = 'test_candace_condense_pixel_output_dir'
</pre><p>This needs to be changed back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>